client/my-sites/plan-price/index.tsx

```diff
@@ -305,15 +304,23 @@ function MultiPriceDisplay( {
 		smallerPrice,
 		currencyCode
 	);
+
+	// Sometimes the US currency symbol is displayed as `US$` instead of just `$`
+	// See: packages/format-currency/README.md (## geolocateCurrencySymbol()) for further details.
+	const hasUslocaleInSymbol = /^US\$$/.test( currencySymbol );
```

Comment:
Is the readability a problem with any other currency symbol? If it is, we might want to handle all of them here instead of just the US. For example, IDR has 2 characters for the symbol as well
![image](https://github.com/Automattic/wp-calypso/assets/65001528/e8830798-c21b-4f12-ba90-e83a0221ad84)

Though it doesn't also have the `$`, so I am not sure. Just wanted to bring up in case you think it should be handled

Reply:
Thanks for reviewing @CodeyGuyDylan!

Hmm, well the readability problem (which was basically not enough vertical spacing) as shown here: ⤵️

![Markup 2023-08-24 at 14 25 42](https://github.com/Automattic/wp-calypso/assets/11078128/ff3976aa-8859-4470-bdcb-07f46d15fb8c)

was occurring on **all** currencies with any substantial length, regardless of how many characters was in the currency symbol. 

So this PR fixes the readability by updating the CSS so that the price doesn't  break into 2 lines, which keeps the vertical spacing looking as it should. But this doesn't necessarily have anything to do with the currency symbol.

But also there was some confusion in the Slack discussion related to this issue as to why the dollar symbol ($) was showing up as "US$".
The "US$" currency symbol is a specific edge case, unique for the USD currency only. If your currency is set to USD and you are physically located in the US, the currency symbol will display as $. If you are located outside the US, the symbol will display as US$.  I only added some styling to it (made the US part of it smaller and raised) just to kind of differentiate it for this specific edge case.  "US$" is still the US dollar.. the same as $, but has the US locale prefixed to it just to prevent confusion if you are located outside the US.  So since the US$ and $ symbols are the same (have the same meaning) I just decided to style the US part of it smaller, I guess to take less space and to de-emphasize it from the $ (dollar sign) a little, and to make it look a little better. You can read about this "US$" vs "$" edge-case here: [geolocateCurrencySymbol()](https://github.com/Automattic/wp-calypso/tree/trunk/packages/format-currency#geolocatecurrencysymbol)
All the other currencies just are what they are, and they don't have an edge case like this.. (AFAIK).
So I don't know, maybe it wasn't necessary.. I just thought it might look better.  If you have any opinions on this, please let me know. I'm open to any improvements or suggestions, of course! 😄

Reply:
Thank you for the very detailed answer, I think you are correct that the others do not need any changing 😄 

I was worried about consistency in style, but it's not many people who are going to be looking at more than one currency so I don't think that's a big deal